frontend/src/css/modules/leaderboard.scss

```diff
@@ -1,9 +1,22 @@
 /* Styles for leaderboard page */
+a:hover {
+	color: blue;
+	text-decoration: underline;
+}
 
-.horizontal-scroll {
-	overflow-x: auto;
+.collapsible-content {
+	box-shadow: 0 5px 11px 0 rgba(0,0,0,.18), 0 4px 15px 0 rgba(0,0,0,.15);
+	overflow: hidden;
+	transition-property: height !important;
+	transition-duration: 1s !important;
+	transition-timing-function: ease-in-out !important;
 }
 
+table.leaderboard-collapse-table {
+	border-collapse: collapse;
+	width: 100%;    
+  	background-color: #f1f1c1;
```

Comment:
```suggestion
```

Reply:
This green color doesn't go well with our eval ai color scheme, keeping it plain won't hurt us.

Reply:
yup yup, done